lib/dialects/mssql/query-generator.js

```diff
@@ -665,7 +665,7 @@ class MSSQLQueryGenerator extends AbstractQueryGenerator {
    * @returns {string}
    */
   _getForeignKeysQueryPrefix(catalogName) {
-    return `${'SELECT ' +
+      return `${'SELECT ' +
```

Comment:
I understand most of the formatting changes you made, this one doesn't make sense tho.